### PR TITLE
Move `sr.robot` module outside controllers dir

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Move the `sr.robot` module out of the controllers dir, as it doesn't make much sense in there.

### CODE REVIEW

File: script/linting/lint [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-PY_ROOT=$(dirname $(dirname  $(dirname $0)))/controllers
+PY_ROOT=$(dirname $(dirname  $(dirname $0)))
```
Comment (PeterJCLaw) [inline script/linting/lint:3]: Perhaps rename this now it's not the _PY_ _ROOT?

(also elsewhere)

File: script/testing/test [hunk 2]
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-cd $(dirname $(dirname  $(dirname $0)))/controllers
+export PYTHONPATH=$(realpath ./stubs):$(realpath ./modules)
 
-export PYTHONPATH=$(realpath ../stub-webots)
-
-python3 -m unittest discover --start-directory sr_controller "$@"
+python3 -m unittest discover --start-directory modules "$@"
```
Comment (PeterJCLaw) [inline script/testing/test:5]: Since we're no longer `cd`ing here, `modules` is, I think, an unreliable name.

I suggest sticking with the `cd` approach, even if the paths are updated.

File: setup.cfg [hunk 3]
```diff
@@ -6,7 +6,8 @@ exclude =
     __pycache__,
     build,
     debian,
-    script
+    script,
+    venv
```
Comment (PeterJCLaw) [inline setup.cfg:10]: This change seems unrelated? Consider cherry-picking to `master`?

Comment (RealOrangeOne) [reply setup.cfg:10]: It seemed related, but mostly during a merge fix-up. Without this `flake8` tried to lint the venv, which naturally took ages.

File: script/testing/test (hunk 2 above)
Comment (RealOrangeOne) [reply script/testing/test:5]: Not sure I see the difference between `cd`ing into the directory vs pointing `unittest` at it?

File: script/linting/lint (hunk 1 above)
Comment (PeterJCLaw) [reply script/linting/lint]: a548fc685acaefb94c23241940668e5a83f61da1

File: script/testing/test (hunk 2 above)
Comment (PeterJCLaw) [reply script/testing/test:7]: 058331824b18530fd728cc7571eba70478755aae
